FrenchQuizApp/Support Files/QuizObject.swift

```diff
@@ -14,7 +14,7 @@ extension QuizObject {
         self.timesCorrect = 0
         self.timesIncorrect = 0
         self.correctInARow = 0
-        self.learned = false
+        self.setAsNotLearned()
```

Comment:
What made you decide to change this to a method?

Reply:
I had multiple ways to do the same thing in this case.  The method to set the learned status of a quiz object already existed, but in the reset method I was changing the value of the learned variable manually instead of using the method.  I thought it would be cleaner to use the method instead of changing the variable directly.